src/United.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="OPL-UK-3.0" name="United
+   Kingdom Open Parliament Licence v3.0" listVersionAdded="">
+      <crossRefs>
+         <crossRef>https://www.parliament.uk/site-information/copyright-parliament/open-parliament-licence/</crossRef>
+      </crossRefs>
+      <standardLicenseHeader>
+      </standardLicenseHeader>
+      <notes>

```

Comment:
a couple hints: the titleText tag is missing and the standardLicenseHeader tag is not needed and can be removed.
you can also probably remove the notes tag, as there are none (that we know of, at this point)

Reply:
@jlovejoy I will look into those issues and get back to you if any clarification needed.